src/utils/permissions.ts

```diff
@@ -2,6 +2,7 @@ import { cacheHandlers } from "../controllers/cache.ts";
 import { botID } from "../module/client.ts";
 import type { Guild } from "../structures/guild.ts";
 import type { Role } from "../structures/role.ts";
+import { RawOverwrite } from "../types/guild.ts";
```

Comment:
Since `RawOverwrite` is not used as a value, it should be imported using `import type` syntax.